Dockerfile.dapper

```diff
@@ -13,12 +13,11 @@ ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} DAPPER_ENV=REPO DAPPE
 RUN rm -f /bin/sh && ln -s /bin/bash /bin/sh
 
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
-    GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
+    GOPATH=/go GOFLAGS="-mod=vendor" PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 
 RUN apt-get -q update && \
     apt-get install -y gcc git curl docker.io && \
-    curl https://storage.googleapis.com/golang/go1.11.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
-    go get -v github.com/rancher/trash && \
+    curl https://storage.googleapis.com/golang/go1.11.11.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
```

Comment:
Could you explain the bump in the commit message?

Reply:
Oh right, I forgot about that detail, thank you @skitt